package.json

```diff
@@ -31,23 +31,26 @@
   },
   "homepage": "https://github.com/qavajs/steps-api#readme",
   "devDependencies": {
-    "@cucumber/cucumber": "^10.8.0",
-    "@qavajs/cli": "^0.39.0",
-    "@qavajs/console-formatter": "^0.7.2",
-    "@qavajs/html-formatter": "^0.17.0",
-    "@qavajs/memory": "^1.7.0",
+    "@cucumber/cucumber": "^11.0.1",
+    "@qavajs/core": "^2.0.0",
+    "@qavajs/validation": "^0.10.0",
+    "@qavajs/memory": "^1.9.0",
+    "@qavajs/console-formatter": "^0.8.0",
+    "@qavajs/html-formatter": "^0.18.1",
     "@types/chai": "^4.3.16",
     "@types/express": "^4.17.21",
     "@types/multer": "^1.4.11",
     "@types/node": "^20.14.9",
     "@types/ws": "^8.5.10",
     "body-parser": "^1.20.2",
-    "express": "^4.19.2",
+    "express": "^4.21.1",
     "prettier": "^3.3.2",
     "ts-node": "^10.9.2",
     "typescript": "^5.5.3",
-    "vitest": "^1.6.0",
-    "@qavajs/validation": "^0.9.0"
+    "vitest": "^1.6.0"
+  },
+  "optionalDependencies": {
```

Comment:
this is not a dependency for api-steps

Reply:
I made to add this dependency to make it work on lunix. Otherwise the test failes with such error: https://github.com/qavajs/steps-api/actions/runs/11519892804/job/32069964473